clerk-frontend/package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "@discipl/abundance-service": "^0.4.1",
     "@discipl/core-ephemeral": "^0.6.2",
-    "@discipl/paper-wallet": "^0.5.0",
+    "@discipl/paper-wallet": "^0.6.0",
+    "canvas-text-wrapper": "^0.10.2",
```

Comment:
This is no longer a needed dependency here, right?

Reply:
True, it has been removed